.github/workflows/test.yml

```diff
@@ -126,7 +126,7 @@ jobs:
       - name: Install rust
         uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
```

Comment:
Do you plan to address [all of the problems](https://github.com/OpenZeppelin/rust-contracts-stylus/issues/426#issuecomment-2527151051) found during the initial investigation?

Reply:
will see.. some part of it

Reply:
In case you don't address all of them, can you update the description from "Resolves 426" to "Towards 426" to indicate there will still be more work?